dotCMS/src/main/java/com/dotcms/publishing/FilterDescriptor.java

```diff
@@ -40,36 +46,62 @@ public class FilterDescriptor implements Comparable<FilterDescriptor> {
 
     private String key;
     private final String title;
+    private final String sort;
```

Comment:
why not the sort is an integer?

Reply:
Met with @jdotcms and confirmed that we're not handling integer values in our YAML files, only BOOL, STRING, or SELECT. See: https://www.dotcms.com/docs/latest/apps-integrations#FileFormat